dev.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e
```

Comment:
i like this. if you want to go full bash strict mode, see http://redsymbol.net/articles/unofficial-bash-strict-mode/